guides/get-started-microservices-on-kubernetes/simple-microservice-part1.adoc

```diff
@@ -0,0 +1,252 @@
+ARG runtime_image=quay.io/wildfly/wildfly-runtime:latest
+FROM ${runtime_image}
+COPY --chown=jboss:root target/server $JBOSS_HOME
+RUN chmod -R ug+rwX $JBOSS_HOME
+----
+
+Build the Docker Image with the following command:
+
+[source,bash,subs="normal"]
+----
+$ podman build -t {docker-image-name}:latest .
+STEP 1/3: FROM quay.io/wildfly/wildfly-runtime:latest
+STEP 2/3: COPY --chown=jboss:root target/server $JBOSS_HOME
+--> cf1b99511a9b
+STEP 3/3: RUN chmod -R ug+rwX $JBOSS_HOME
+COMMIT {docker-image-name}:latest
+--> e1ab6e80ed20
+Successfully tagged localhost/{docker-image-name}:latest
+e1ab6e80ed20c3619a7e859f03c71f33b79a4d292f971ed83e7484f4779121d8
+----
+
+NOTE: As you can see, we used link:https://podman.io/[Podman], but you can use link:https://www.docker.com/[Docker] as well, depending on your preference or what you have installed on your PC; to do that, replace *podman* with *docker* in the previous command (and in the ones that will follow);
+
+=== Run the Docker Image locally
+
+To check that our brand-new `{docker-image-name}:latest` Docker Image works as expected, run the following command:
+
+[source,bash,subs="normal"]
+----
+$ podman run --rm -p 8080:8080 -p 9990:9990 --name={docker-image-name} {docker-image-name}
+...
+16:14:49,477 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-5) WFLYUT0006: Undertow HTTP listener default listening on 0.0.0.0:8080
+...
+16:14:49,677 INFO  [org.jboss.weld.deployer] (MSC service thread 1-4) WFLYWELD0003: Processing weld deployment ROOT.war
+...
+16:14:50,403 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 61) RESTEASY002225: Deploying jakarta.ws.rs.core.Application: class org.wildfly.examples.GettingStartedApplication
+16:14:50,426 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 61) WFLYUT0021: Registered web context: '/' for server 'default-server'
+16:14:50,451 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 35) WFLYSRV0010: Deployed "ROOT.war" (runtime-name : "ROOT.war")
+16:14:50,459 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
+16:14:50,460 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://0.0.0.0:9990/management
+16:14:50,460 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0054: Admin console is not enabled
+16:14:50,461 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: WildFly Full 31.0.0.Final (WildFly Core 23.0.1.Final) started in 1101ms - Started 280 of 378 services (149 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
+----
+
+As you can, see the applications started, and it's ready to serve requests on port `8080`;
+
+NOTE: we explicitly mapped port `8080` in the container to port `8080` on the HOST (your PC); we also mapped port `9990` in the container to port `9990` on the HOST to expose the Liveness, Readiness and Startup Probes;
+
+=== Check the application
```

Comment:
I'm not sure about duplicating the steps to check the when they're the same as above. Maybe it could be mentioned that it can be checked the same way as above (+link)

Reply:
@mnovak1 thank you for the suggestion: the procedure for checks is the same as before and I replaced it with a link